api/script/db/tables.sql

```diff
@@ -0,0 +1,31 @@
+-- route
```

Comment:
need a doc about how to use it

Reply:
I rename table.sql to schema.sql, modified the content and add some description.
This integration is specified in docker-compose, use the `docker-compose` command in the compose directory.